### PR TITLE
CCXDEV-7025 Update the SCA configuration docs

### PR DESCRIPTION
**Pull request by xJustin**
Update to the SCA docs to prepare for 4.10 GA

https://issues.redhat.com/browse/CCXDEV-7025

Versions: enterprise-4.10 + 

Preview:
https://deploy-preview-40689--osdocs.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/insights-operator-simple-access.html

SME: tremes 
QE: JoaoFula

### CODE REVIEW

**Comment by tremes** [comment]
LGTM

File: support/remote_health_monitoring/insights-operator-simple-access.adoc [hunk 1]
```diff
@@ -1,20 +1,19 @@
...
 
-Insights Operator imports SCA certificates every 8 hours by default, but can be configured or disabled using the `support` secret in the `openshift-config` namespace.
+Insights Operator imports simple content access certificates every 8 hours by default, but can be configured or disabled using the `support` secret in the `openshift-config` namespace.
 
-In {product-title} 4.9, this feature is in Technology Preview and must be enabled using the `TechPreviewNoUpgrade` Feature Set. See xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc[_Enabling OpenShift Container Platform features using FeatureGates_] for more information.
+For more information about simple content access see link:https://access.redhat.com/documentation/en-us/subscription_central/2021/html-single/getting_started_with_simple_content_access/index#assembly-about-simplecontent[_About simple content access_] in the Red Hat Subscription Central documentation.
```
Comment (mburke5678) [inline support/remote_health_monitoring/insights-operator-simple-access.adoc:13]: s/access see/access, see

File: modules/insights-operator-configuring-sca.adoc [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 
 [id="insights-operator-configuring-sca_{context}"]
-= Configuring Simple Content Access import interval
+= Configuring simple content access import interval
 
-You can configure how often the Insights Operator imports the RHEL Simple Content Access (SCA) certificates using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you may want to shorten this interval if you update your SCA configuration in Red Hat Subscription Management.
+You can configure how often the Insights Operator imports the simple content access certificates using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you may want to shorten this interval if you update your simple content access configuration in Red Hat Subscription Management.
```
Comment (mburke5678) [inline modules/insights-operator-configuring-sca.adoc:9]: I know this is existing text, but can you s/may/might? Or maybe better, _but you can shorten..._.  
Or even two sentences? 
_The certificate import normally occurs every 8 hours. You can shorten..._

**Comment by mburke5678** [comment]
@xJustin A few nits. Otherwise LGTM

File: support/remote_health_monitoring/insights-operator-simple-access.adoc (hunk 1 above)
Comment (xJustin) [reply support/remote_health_monitoring/insights-operator-simple-access.adoc]: updated

File: modules/insights-operator-configuring-sca.adoc (hunk 2 above)
Comment (xJustin) [reply modules/insights-operator-configuring-sca.adoc]: Updated -made more sense to keep it one sentence- , but you can shorten...

**Comment by JoaoFula** [comment]
LGTM

**Comment by xJustin** [comment]
@lpettyjo can you please try to cherrypick this to 4.10?

**Comment by mburke5678** [comment]
@xJustin Should you make the [same name change](https://github.com/openshift/openshift-docs/pull/40689/files#diff-7c09a5ccc63f204d75ed89dd359a340a43551a4a790ae36fb9c64f6c5149211bR483) to the [topic in 4.9](https://docs.openshift.com/container-platform/4.9/support/remote_health_monitoring/insights-operator-simple-access.html), so that folks don't think the topic was removed??

**Comment by xJustin** [comment]
@mburke5678 I suppose we can do that. Would I have to open another Jira issue or could I just mention this PR? It would only be the topic change and maybe the module headings as the changes in this PR only apply to 4.10+.

**Comment by mburke5678** [comment]
I should think you can just mention this PR in a new PR. I think just updating the heading would make sense. (That would drop the need for QE review, I believe.) Thanks!

**Comment by xJustin** [comment]
@mburke5678 https://github.com/openshift/openshift-docs/pull/41677

**Pull request merged**